src/components/navMenu/navMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import { dcNavLinks } from "@/data/dcNavLinks";
+import { NavMenuProps } from "@/types/NavMenuProps";
```

Comment:
You're not using `NavMenuProps` in this file. It's also okay to keep interfaces or types within the same file as the components that use them unless if the interface or type is generic.